package/MDAnalysis/coordinates/src/dcd.c

```diff
@@ -1,949 +0,0 @@
-/* -*- Mode: C; tab-width: 4; indent-tabs-mode:nil; -*- */
-/* vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 */
-/*
-
-   MDAnalysis --- http://www.mdanalysis.org
-   Copyright (c) 2006-2016 The MDAnalysis Development Team and contributors
-   (see the file AUTHORS for the full list of names)
-
-   Released under the GNU Public Licence, v2 or any higher version
-
-   Please cite your use of MDAnalysis in published work:
-
-   R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
-   D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
-   MDAnalysis: A Python package for the rapid analysis of molecular dynamics
-   simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
-   Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
-
-   N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
-   MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
-   J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
-
-*/
-
-
-/* I have to fix error handling in these functions */
-
-#include <Python.h>
-#include <numpy/arrayobject.h>
-
-#include "readdcd.h"
-
-typedef struct {
-  fio_fd fd;
-  fio_size_t header_size;
-  int natoms;
-  int nsets;
-  int setsread;
-  int istart;
-  int nsavc;
-  double delta;
-  int nfixed;
-  int *freeind;
-  float *fixedcoords;
-  int reverse;
-  int charmm;
-  int first;
-  int with_unitcell;
-} dcdhandle;
-
-
-int jump_to_frame(dcdhandle *dcd, int frame);
-
-static PyObject *
-__write_dcd_header(PyObject *self, PyObject *args)
-{
-  /* At this point we assume the file has been opened for writing */
-  PyObject *temp = NULL;
-  dcdhandle *dcd = NULL;
-  fio_fd fd;
-  int rc = 0;
-  int natoms = 0;
-  const char *remarks = "DCD";
-  int istart = 0;             /* starting timestep of DCD file                  */
-  int nsavc = 1;              /* number of timesteps between written DCD frames */
-  double delta = 1.0;         /* length of a timestep                           */
-  int with_unitcell = 1;      /* contains unit cell information (charmm format) */
-  int charmm = DCD_IS_CHARMM; /* charmm-formatted DCD file                      */
-  if (with_unitcell)
```

Comment:
The old code was able to write DCDs without unitcell information, the new one isn't.

It's probably not a big deal but it is a regression.